packages/web/components/bridge/use-bridges-supported-assets.ts

```diff
@@ -56,14 +56,6 @@ export const useBridgesSupportedAssets = ({
             // This causes slow UX even though there's a
             // query that the user can use.
             retry: false,
-
-            // prevent batching so that fast routers can
-            // return requests faster than the slowest router
-            trpc: {
-              context: {
-                skipBatch: true,
-              },
-            },
```

Comment:
I realized there was a race condition when the initial from chain was getting set. We wanted to pick EVM chains first, but if a query landed with just cosmos chains, the first cosmos chain would be picked.
Hopefully, if we send all the supported assets queries as a single batch, we'd more reliably get the chains we want to sort and set initially